sample/build.gradle

```diff
@@ -36,18 +36,18 @@ fladle {
     smartFlankGcsPath = "gs://test-lab-yr9w6qsdvy45q-iurp80dm95h8a/flank/test_app_android.xml"
     configs {
         oranges {
-            useOrchestrator = false
-            testTargets = [
+            useOrchestrator.set(false)
```

Comment:
this change should not be necesary. If you revert, does it still work? I would like to keep backward compatibility with the groovy dsl.

Reply:
I am afraid it is necessary. Without it, tests are failing. What is more, if you add `fullJunitResult` (which on current master branch is `Property<Boolean>`) to `sample` `build.gradle` like:
```gradle
fladle {
    ...
    configs {
        oranges {
            ...
            flakyTestAttempts = 3
            fullJunitResult = true
        }
        ...
    }
    ...
}
```
it fails with:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/gogo/fladle/sample/build.gradle' line: 44

* What went wrong:
A problem occurred evaluating project ':sample'.
> Cannot cast object 'true' with class 'java.lang.Boolean' to class 'org.gradle.api.provider.Property'
```

Reply:
This is bug in Gradle: https://github.com/gradle/gradle/issues/9987 😢